ci_framework/roles/openshift_login/tasks/try_login.yml

```diff
@@ -34,8 +34,9 @@
   environment:
     PATH: "{{ cifmw_path }}"
     KUBECONFIG: "{{ cifmw_openshift_login_kubeconfig }}"
-  ansible.builtin.command:
-    cmd: >-
+  ci_script:
+    output_dir: "{{ cifmw_openshift_login_basedir }}/artifacts"
```

Comment:
Hey @adrianfusco I wouldn't be surprised if you answered this question somewhere already.
Why are you replacing `ansible.builtin.command` with `ci_script` plugin? IIUC we're not gathering any logs here, hence there is no immediate need to change that.

Additionally, this piece of code uses `environment`. I checked the code [1] but I don't see passing of environment variables there. @pablintino can `ci_script` be used with `environment` variables?

[1]: https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/plugins/action/ci_script.py

Reply:
Hi @dasm, great thought. I quickly tested this and can confirm it's fine.

Environment sets the shell environment vars so the ci_script module gets the support for free.

```
- name: test ci_script
  hosts: localhost
  tasks:
    - name: test ci_script
      environment:
        TESTVAR: "hello"
      ci_script:
        output_dir: "/Users/ldenny/Code/ci/ci-framework/plugins/action/test"
        script: >-
          echo $TESTVAR ;
          env | grep TESTVAR
```

```
changed: [localhost] => {
    "changed": true,
    "rc": 0,
    "stderr": "",
    "stderr_lines": [],
    "stdout": "hello\nTESTVAR=hello\n",
    "stdout_lines": [
        "hello",
        "TESTVAR=hello"
    ]
}
```

Reply:
Neat, thanks!